### ISSUE TITLE
AppRTC like RTCMultiConnection does not work in firefox

### ISSUE THREAD

**Issue by nespaona** [muaz-khan/RTCMultiConnection#534]
Hello since the last update of firefox, this demo is not working

https://www.webrtc-experiment.com/RTCMultiConnection/AppRTC-Look.html

In Google Chrome it works without problems.

Please, you can solve this problem and update the RTCMultiConnection.js

Thank you

**Comment by nhoctanker** [comment]
Yes . Current version 2 not working on fire fox please fix thank you

**Comment by hassanraza05128** [comment]
Not Working with Firefox .

**Comment by utkarsh2k2** [comment]
Same here.. please look into the issue
